cli/command/container/stats.go

```diff
@@ -175,12 +175,12 @@ func runStats(dockerCli *command.DockerCli, opts *statsOptions) error {
 		for _, c := range cStats.cs {
 			cErr := c.GetError()
```

Comment:
oh, any reason you didn't put this in the `if` initialization? I think `cErr` is only used inside the `if`, so can be scoped to that (see my example)